docs/assembly-control-unit.md

```diff
@@ -1,164 +1,274 @@
+    * Place CNC shield on the Arduino, push the CNC pins  into the blue header sockets on the Arduino until it is fully connected.
+<IMG ALT="Adding Shield" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+  1. Add the Stepper Motor Driver Boards
+    * Orient each of the drivers. **The A4988 driver has a small potentiometer on the BOTTOM center of its circuit board** The potentiometer looks like a small phillips head screw.
+    * Insert the stepper motors driver boards into the X, Y, and Z slots on the shield.
+<IMG ALT="Adding Stepper Motor Driver Boards" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+  1. Set Enable Pin to Ground
+    * Place 1 female-female jumper lead on the enable and ground pins to
+  enable the board.
+  <IMG ALT="Set Enable to Ground" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+  1. Add Stepper Motor Cables
+    * Connect the stepper motors cables
+<IMG ALT="Adding Stepper Motor Cables" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+  1. Connect the Power Supply (but don't turn it on)
+    * Take the Power Supply (CU-SPS) and the connect to the 12v power connection on the lower left of the shield should be directly above the Arduino power jack also on the lower left.
+    * Be sure to double check that
+       * you have installed the drivers according to the manufacturer's instructions
+       * all the drivers are firmly seated in the sockets, and that none of the heat sinks shifted during installation.
+    * **Double check the polarity otherwise the power supply may damage the stepper motor driver boards** <br>
+<IMG ALT="Connecting power" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
 
-### Installing the SD card
+Note: Keep it turned off. At this point you do not turn on the power to the shield. If a stepper motor is not connected to the shield, it can cause the stepper motors drivers to overheat and become damaged. ALWAYS connect stepper motors to the CNC Shield before powering it on to test or use it.
 
-1. Once complete, remove the SD card from your computer, and insert into your Raspberry Pi.
-1. Note that the SD card extends beyond the edge of the Pi's circuit board. The Pi is very delicate with the card inserted, so take care.
+## Stepper Motor Calibration
 
-### Installing the touchscreen
+Before we continue with assembly we should tune the potentiometers and test the stepper motor wiring.
 
-1. Gently slide the screen's female headers over the Pi's GPIO pins.
-1. The screen should line up with the Pi's board and the end of the headers should also coincide.
-1. NOTE: We will need to remove this again to put the GPIO pins in place
+We will need the:
+- Raspberry Pi that is setup
+- Arduino that is setup
+- **Pi power cable?**
+- Peristaltic pumps with stepper motors
+- Small phillips head screw driver
 
-## Arduino Setup
+### Potentiometer tuning
```

Comment:
We probably want to move this to the end after everything is assembled. Otherwise assemblers will need to set up preliminary wiring, which will only add complexity for no real gain.